src/components/Todo/Todo.module.css

```diff
@@ -1,49 +1,127 @@
-.wrap {
-    width: 350px;
-    height: 550px;
-    margin: auto;
-    display: flex;
-    flex-flow: column nowrap;
-    align-items: center;
-    justify-content: space-around;
+.todo {
+  position: absolute;
+  width: 700px;
+  height: 644px;
+  margin: auto;
+  background: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)